### PR TITLE
Issue #2544: Design review on view and manage domains, org feature - [HOTGOV]

### LINKED ISSUE #2544: Design Review: View and Manage domains
### Issue description

Domain management should now be complete, including the domain table, navigation, and both read-only and editable views of the domain management pages. Excluded from this review is anything for organization details, requests, etc.

### Acceptance criteria

- [x] Design reviews the domain management table and pages
- [x] Document any changes needed or new additions in a document, including with screenshots so it is easy to follow
- [x] Discuss changes with dev and determine which are small email to be done as part of this ticket 
- [x] Work with dev to create new tickets for larger changes 

### Additional context

**Working with dev:** Dev effort on this shouldn't be more than a "2" (2 full work days). 
- Dev is responsible for answering questions from design, making minor changes (such as those that we would normally make on a PR).
- Anything that would take more than 4 hours to complete alone should be it's own ticket. 
- If the amount of changes desired would result in the dev spending more than a full day on it, make a dev ticket for items where reasonable (communicate with Alysia).

NOTE: This design review does not include sending/testing email notifications to domain managers. That will be tested separately once that work is implemented.

### Temporary Slack channel
Discussions for this design review should take place in this Slack channel: [#2544-design-review](https://dhscisa.enterprise.slack.com/archives/C07HJTNV066)

### Tickets covered in this design review

- blocked by #2493 
- blocked by #2460 
- blocked by #2348 
- blocked by #2354 
- blocked by #2352

### PR DESCRIPTION
**Pull request by rachidatecs**
## Ticket

Resolves #2544 

## Changes

- Enhance zip field
- Change vertical alignment on td

#### Satisfied acceptance criteria and met development standards

- [ ] Met the acceptance criteria, or will meet them in a subsequent PR
- [ ] Created/modified automated tests
- [ ] Added at least 2 developers as PR reviewers (only 1 will need to approve)
- [ ] Messaged on Slack or in standup to notify the team that a PR is ready for review
- [ ] Changes to “how we do things” are documented in READMEs and or onboarding guide
- [ ] If any model was updated to modify/add/delete columns, makemigrations was ran and the associated migrations file has been commited.

#### Ensured code standards are met (Original Developer)

- [ ] All new functions and methods are commented using plain language
- [ ] Did dependency updates in Pipfile also get changed in requirements.txt?
- [ ] Interactions with external systems are wrapped in try/except
- [ ] Error handling exists for unusual or missing values

#### Validated user-facing changes (if applicable)

- [ ] New pages have been added to .pa11yci file so that they will be tested with our automated accessibility testing
- [ ] Checked keyboard navigability
- [ ] Tested general usability, landmarks, page header structure, and links with a screen reader (such as Voiceover or ANDI)
- [ ] Add at least 1 designer as PR reviewer

#### Reviewed, tested, and left feedback about the changes

- [ ] Pulled this branch locally and tested it
- [ ] Reviewed this code and left comments
- [ ] Checked that all code is adequately covered by tests
- [ ] Made it clear which comments need to be addressed before this work is merged
- [ ] If any model was updated to modify/add/delete columns, makemigrations was ran and the associated migrations file has been commited.

#### Ensured code standards are met (Code reviewer)

- [ ] All new functions and methods are commented using plain language
- [ ] Interactions with external systems are wrapped in try/except
- [ ] Error handling exists for unusual or missing values
- [ ] (Rarely needed) Did dependency updates in Pipfile also get changed in requirements.txt?

#### Validated user-facing changes as a developer

- [ ] New pages have been added to .pa11yci file so that they will be tested with our automated accessibility testing
- [ ] Checked keyboard navigability
- [ ] Meets all designs and user flows provided by design/product
- [ ] Tested general usability, landmarks, page header structure, and links with a screen reader (such as Voiceover or ANDI)

- [ ] Tested with multiple browsers, the suggestion is to use ones that the developer didn't (check off which ones were used)
  - [ ] Chrome
  - [ ] Microsoft Edge
  - [ ] FireFox
  - [ ] Safari

- [ ] (Rarely needed) Tested as both an analyst and applicant user

**Note:** Multiple code reviewers can share the checklists above, a second reviewers should not make a duplicate checklist

#### Verified that the changes match the design intention

- [ ] Checked that the design translated visually
- [ ] Checked behavior
- [ ] Checked different states (empty, one, some, error)
- [ ] Checked for landmarks, page heading structure, and links
- [ ] Tried to break the intended flow

#### Validated user-facing changes as a designer

- [ ] Checked keyboard navigability
- [ ] Tested general usability, landmarks, page header structure, and links with a screen reader (such as Voiceover or ANDI)

- [ ] Tested with multiple browsers (check off which ones were used)
  - [ ] Chrome
  - [ ] Microsoft Edge
  - [ ] FireFox
  - [ ] Safari

- [ ] (Rarely needed) Tested as both an analyst and applicant user

### CODE REVIEW

**Comment by AnnaGingle** [comment]
@gabydisarli Can you take this one? I'm not as familiar with the hint text patterns as you.

**Comment by AnnaGingle** [comment]
The vertical alignment on the td looks good. I'm wondering if we can also fix it on the th in this PR?
Screenshot from sandbox:
Screenshot 2024-08-27 at 11 17 28 AM

Design th alignment:
TableHeader

**Comment by AnnaGingle** [comment]
The zip code field looks good to me, The ANDI output is as expected.
The hint text does not meet the minimum AA contrast ratio (4.5:1).  @gabydisarli What do you think? 
Screenshot 2024-08-27 at 11 22 32 AM


On the DNS name servers page the hint text color is #71767a which passes all the contrast levels except AAA regular text
Screenshot 2024-08-27 at 11 30 05 AM
Screenshot 2024-08-27 at 11 31 21 AM

**Comment by gabydisarli** [comment]
> The zip code field looks good to me, The ANDI output is as expected. The hint text does not meet the minimum AA contrast ratio (4.5:1). @gabydisarli What do you think? Screenshot 2024-08-27 at 11 22 32 AM
> 
> On the DNS name servers page the hint text color is #71767a which passes all the contrast levels except AAA regular text Screenshot 2024-08-27 at 11 30 05 AM Screenshot 2024-08-27 at 11 31 21 AM

This is because the background color is wrong. When the UI switches to the interior pages (not Domains, Domain requests, Members), the background color should be white not the off white color. @rachidatecs the background color on Org pages (Org info and Senior official) should be white.

**Comment by abroddrick** [comment]
@rachidatecs this is not ready for dev review, correct?

**Comment by rachidatecs** [comment]
> @rachidatecs this is not ready for dev review, correct?

I suggest waiting on @gabydisarli to approve first

**Review by AnnaGingle — APPROVED** [review]

File: src/registrar/templates/portfolio_requests.html [hunk 1]
```diff
@@ -4,7 +4,12 @@
 
 {% block title %} Domain requests | {% endblock %}
 
+{% block wrapper_class %}
+  {{ block.super }} dashboard--grey-1
+{% endblock %}
+
 {% block portfolio_content %}
+<div id="main-content">
```
Comment (asaki222) [inline src/registrar/templates/portfolio_requests.html:12]: Is main-content used mostly for organization of the templates? I don't see any styling references within the codebase.

Comment (rachidatecs) [reply src/registrar/templates/portfolio_requests.html:12]: It's used as a target for the skip nav functionality, which is an accessibility requirement

Comment (asaki222) [reply src/registrar/templates/portfolio_requests.html:12]: Thank you for clarifying!

**Review by asaki222 — APPROVED** [review]
LGTM!

**Review by abroddrick — APPROVED** [review]
Looks great once Todos are removed

**Pull request merged**